### PR TITLE
Recurse directories when gathering FSH definitions

### PR DESCRIPTION
**Pull request by peturv**
Added a flag to allow for recursion into subdirectories when gathering FSH definitions to allow for a directory based structure for larger projects.

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -155,3 +160,9 @@ function getVersion(): string {
   }
   return 'unknown';
 }
+
+function getFilesRecursive(dir: string): string[] {
+  return fs.statSync(dir).isDirectory()
+    ? [].concat(...fs.readdirSync(dir, 'utf8').map(f => getFilesRecursive(path.join(dir, f))))
+    : [dir];
```
Comment (cmoesel) [inline src/app.ts]: This is stylistic, but it seems like there is a lot happening here in a single statement.  For readability, you might consider a more traditional `if/else` clause.

**Comment by cmoesel** [comment]
Thanks again for the PR!  As noted in #239, I'd ask that you go ahead and remove the flag, making recursive traversal the norm.  I also made one comment that is more of a stylistic preference.  I'll accept the PR either way, but ask that you consider it.  Thanks!

File: src/app.ts (hunk 1 above)
Comment (peturv) [reply src/app.ts]: Would this be better?
```suggestion
    function getFilesRecursive(dir: string): string[] {
  if(fs.statSync(dir).isDirectory()) {
    const ancestors = fs.readdirSync(dir, 'utf8').map(f => getFilesRecursive(path.join(dir, f)));
    return [].concat(...ancestors);
  } else {
    return [dir];
  }
```

Comment (peturv) [reply src/app.ts]: Would this be better? 
```
  if(fs.statSync(dir).isDirectory()) {
    const ancestors = fs.readdirSync(dir, 'utf8').map(f => getFilesRecursive(path.join(dir, f)));
    return [].concat(...ancestors);
  } else {
    return [dir];
  }
```

**Comment by cmoesel** [comment]
I don't know why, but GitHub CI is stuck.  I've tested this locally and confirmed it is good.  Using my administrator super-powers to force the merge.

**Pull request merged**